.github/workflows/software-environments-nightly.yml

```
@@ -0,0 +1,76 @@
+name: Create software environments
```

We want this workflow to have a unqiue name. It currently has the same name as the `.github/workflows/software-environments-stable.yml` workflow 

```suggestion
name: Create nightly Coiled software environments
```

---

While we're at it, can we change the name in the other workflow to be `Create Coiled software environments` instead of `Create software environments`?